src/VolleyManagement.Backend/VolleyManagement.UI/Views/Home/Index.cshtml

```diff
@@ -5,4 +5,4 @@
 <h2>Index</h2>
 <li>@Html.ActionLink("MVC", "Index", "Tournaments", new { area = "Mvc" }, new { })</li>
 
-<li>@Html.ActionLink("Web API", "Index", new { area = "WebApi" })</li>
```

Comment:
How does this change relate to the task?

Reply:
This doesn't.